src/reformatters/dwd/icon_eu/forecast/template_config.py

```diff
@@ -0,0 +1,593 @@
+                    long_name="2 metre relative humidity",
+                    units="%",
+                    step_type="instant",
+                    comment="Relative humidity at 2m above ground. Other short_names used for this parameter: rh, 2r, r.",
+                    standard_name="relative_humidity",
+                ),
+                internal_attrs=DwdIconEuInternalAttrs(
+                    grib_element="relhum_2m",
+                    keep_mantissa_bits=default_keep_mantissa_bits,
+                ),
+            ),
+            DwdIconEuDataVar(
+                name="soil_water_runoff",
+                encoding=encoding_float32_default,
+                attrs=DataVarAttrs(
+                    short_name="watr",
+                    long_name="Soil water runoff",
+                    units="kg m-2",
+                    step_type="accum",
+                    comment="Soil water runoff (accumulated since model start)",
+                ),
+                internal_attrs=DwdIconEuInternalAttrs(
+                    grib_element="runoff_g",
+                    keep_mantissa_bits=default_keep_mantissa_bits,
+                    deaccumulate_to_rate=True,
+                ),
+            ),
+            DwdIconEuDataVar(
+                name="surface_water_runoff",
+                encoding=encoding_float32_default,
+                attrs=DataVarAttrs(
+                    short_name="watr",
+                    long_name="Surface water Runoff",
+                    units="kg m-2",
+                    step_type="accum",
+                    comment=(
+                        "Surface water runoff from interception and snow reservoir and from"
+                        " limited infiltration rate. Sum over forecast."
+                    ),
+                ),
+                internal_attrs=DwdIconEuInternalAttrs(
+                    grib_element="runoff_s",
+                    keep_mantissa_bits=default_keep_mantissa_bits,
+                    deaccumulate_to_rate=True,
+                ),
+            ),
+            DwdIconEuDataVar(
+                name="temperature_2m",
+                encoding=encoding_float32_default,
+                attrs=DataVarAttrs(
+                    short_name="t2m",
+                    long_name="2 metre temperature",
+                    units="K",
+                    step_type="instant",
+                    comment=(
+                        "Temperature at 2m above ground, averaged over all tiles of a grid point. Different agencies use different short_names for this parameter: ECMWF: 2t; NOAA & DWD: t2m."
+                    ),
+                    standard_name="air_temperature",
+                ),
+                internal_attrs=DwdIconEuInternalAttrs(
+                    grib_element="t_2m",
+                    keep_mantissa_bits=default_keep_mantissa_bits,
+                ),
+            ),
+            DwdIconEuDataVar(
+                name="total_precipitation",
```

Comment:
```suggestion
                name="precipitation_surface",
```
to match our other datasets